examples/kustomize.jsonnet

```diff
@@ -4,6 +4,10 @@ local kp =
       common+: {
         namespace: 'monitoring',
       },
+
+      kubernetesControlPlane+: {
+        kubeProxy: true,
+      },
```

Comment:
Please remove this. We don't want to ship kubeProxy enabled by default (it shouldn't appear in `manifests/`).